zeppelin-interpreter/src/test/java/org/apache/zeppelin/interpreter/InterpreterTest.java

```diff
@@ -31,7 +30,37 @@
   public void testDefaultProperty() {
     Properties p = new Properties();
     p.put("p1", "v1");
-    MockInterpreterA intp = new MockInterpreterA(p);
+    Interpreter intp = new Interpreter(p) {
```

Comment:
instead of duplicating this would it be less error prone to have a common interpreter test java file to define this?